### ISSUE TITLE
edit temperature sensor 12-HD Max

### ISSUE THREAD

**Issue by javifly** [kendallgoto/ilo4_unlock#26]
one of my temperature sensors has broken down and is showing very high values, it is the "12-HD Max" because of this the server is shutting down due to thermal protection. There would be a way to disable this sensor or raise the critical temperature value about 10 degrees ...? thanks

**Comment by kendallgoto** [comment]
yes, you can disable the sensor from affecting calculations, with `fan t ## off`. the number can be looked up with `fan info t` - it might be 12 as well, but not sure.

**Comment by javifly** [comment]
Does the fan command disable temperature sensors or fans?

**Comment by kendallgoto** [comment]
both

**Comment by javifly** [comment]
maybe sensor number 11 ??

<img width="515" alt="image" src="https://github.com/user-attachments/assets/22cc81cb-376e-467f-9e89-9090486aeff5">

<img width="333" alt="image" src="https://github.com/user-attachments/assets/f420bd3f-b3c8-4ae6-9440-641b34e4a0c0">

**Comment by javifly** [comment]
see the command


you can increase the tolerance from 60 to 63 and make it not shut down.


`
fan t 11 caut 65
`

**Comment by javifly** [comment]
i have executed the command and i think it worked. now on the ilo web site it tells me that the caution temperature is 62 degrees instead of 60 degrees.

is this change permanent? or if the power goes out is it lost?

<img width="330" alt="image" src="https://github.com/user-attachments/assets/e263783d-89df-4a90-9d02-9b5020b3cd01">

**Comment by javifly** [comment]
As soon as I restarted the Ilo, the value returned to 60. 

is this change permanent? or if the power goes out is it lost?

**Comment by kendallgoto** [comment]
yeah that looks good. 

and no, no settings are persistent. they have to be reapplied whenever power is lost to the iLO

**Comment by javifly** [comment]
Thanks!
